nuxeo-webdav/src/main/java/org/nuxeo/ecm/webdav/ExceptionHandler.java

```diff
@@ -55,6 +57,10 @@ public Response toResponse(Exception e) {
                 log.error(e, e);
             }
             // msg = "Error " + status + "\n" + e.getMessage() + "\n" + sw;
+        } else if (e.getCause() instanceof RecoverableClientException
+                && ("QuotaExceededException".equals(e.getCause().getClass().getSimpleName()))) {
```

Comment:
`QuotaExceededException.class.getSimpleName()` would be better than a raw string if its available in the classpath.

Reply:
Indeed but it's not, Quota and WebDAV are two independent modules... I could have changed `QuotaExceededException` into a subclass of a more generic new `InsufficientStorageException` that would be added in the core but that's more changes than I wanted to do for a Windows WebDAV bugfix.